jwst/outlier_detection/utils.py

```diff
@@ -276,3 +298,9 @@ def flag_model_crs(image, blot, snr):
     match_nans_and_flags(image)
 
     log.info(f"{np.count_nonzero(cr_mask)} pixels marked as outliers")
+
+
+def _make_blot_model(input_model, blot):
+    blot_model = type(input_model)()
+    blot_model.data = blot
```

Comment:
This should work.  But can we add an update call here, so it gets the top-level metadata?

Reply:
whoops, yeah, simply forgot this

Reply:
done